snippets/rust.json

```diff
@@ -29,10 +29,17 @@
         ],
         "description": "Insert unreachable!"
     },
+    "print": {
+        "prefix": "print",
+        "body": [
+            "print!($1)$0"
+        ],
+        "description": "Insert print!"
+    },
     "println": {
         "prefix": "println",
         "body": [
-            "println!(\"${1:{\\}}\", $2)$0"
+            "println!($1)$0"
```

Comment:
I don't think this is a great fix since I think it is nice to have the `"", ` there for the user and have the tab stops for inside and outside. However, you're probably right that having the `{}` is not necessary.